content/server/ha.md

```diff
@@ -0,0 +1,214 @@
+---
+date: 2000-01-01T00:00:00+00:00
+title: HA
+author: tphoney
+weight: 21
+toc: true
+description: |
+  Configure Drone server HA.
+---
+
+<div class="alert">
+HA (high availability) for drone is in Beta and may not be suitable for production workloads. Furthermore this is a community effort and is not subject to support services or service level agreements at this time.
+</div>
+
+Drone is a now capable of being a highly available service that is resilient to failures.
+
+# Overview
+
+![HA diagram](/ha_server.png)
+
+There are two new components that are required to make drone highly available.
+
+- The [HAProxy](https://www.haproxy.org/) load balancer. Used to load balance the drone server UI, and also to balance requests from the drone runners to the drone servers.
```

Comment:
I suspect most people using HA will be using their cloud vendor's load balancer such as the AWS Elastic Load Balancer.  It may be worth clarifying that a Load Balancer is required, but not necessarily HAProxy.

> The Load Balancer. Used to load balance the drone server UI, and also to balance requests from the drone runners to the drone servers.  The sample configuration in this article uses HAProxy, however, any standard Load Balancer or reverse proxy should work.